### PR TITLE
Segmented control views

### PR DESCRIPTION
**Pull request by Dagmawit-AHG**

### CODE REVIEW

File: FlightAppInSwiftUI/View/RoundTripView.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+struct RoundTripView: View {
+    @State private var fromRoundTripText: String = String()
+    @State private var toRoundTripText: String = String()
+    @State private var departureDate = Date.now
+    @State private var returnDate = Date.now
+    var body: some View {
+        ScrollView {
+            VStack (spacing: 30){
+                ZStack {
```
Comment (canakyildz) [inline FlightAppInSwiftUI/View/RoundTripView.swift:18]: from this line to 27th line, you could notice that there is a repetitive code. Could you please refactor it? Maybe you could create a opaque type view returning function that takes all the same components and modifiers applied repetitively in the same area.

File: FlightAppInSwiftUI/View/RoundTripView.swift [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                    .frame(minWidth: 0, maxWidth: .infinity, minHeight: 0, maxHeight: .infinity, alignment: .trailing).padding()
+                TextField("To",text: $toRoundTripText)
+                    .frame(height: 50)
+                    .padding()
+                    .border(.clear)
+                    .overlay(RoundedRectangle(cornerRadius: 5)
+                        .strokeBorder(Color.accentColor))
+            }
+                HStack(spacing: 30) {
+                ZStack {
```
Comment (canakyildz) [inline FlightAppInSwiftUI/View/RoundTripView.swift]: same here to 59, if repetitive -> refactor after implementation

File: FlightAppInSwiftUI/View/RoundTripView.swift (hunk 1 above)
Comment (Dagmawit-AHG) [reply FlightAppInSwiftUI/View/RoundTripView.swift:18]: Done 👍
I've removed the repetitive modifier code and put them in a new swift file that contains functions that return opaque type views

File: FlightAppInSwiftUI/View/RoundTripView.swift (hunk 2 above)
Comment (Dagmawit-AHG) [reply FlightAppInSwiftUI/View/RoundTripView.swift]: Fixed 👍

**Review by canakyildz — APPROVED** [review]

**Pull request merged**